setup.cfg

```
@@ -35,22 +35,21 @@ zip_safe = False
 include_package_data = True
 packages = find:
 install_requires =
-    ghga-service-chassis-lib[api,mongo-connect]==0.15.0
+    ghga-service-chassis-lib[api]==0.15.1
```

Möchtest du vielleicht auf den neuen release warten?

```suggestion
    ghga-service-chassis-lib[api]==0.15.2
```

---

Du brauchst doch requests in production?

---

https://github.com/ghga-de/ghga-service-chassis-lib/actions/runs/3480482584